scormxblock/static/js/src/studio.js

```diff
@@ -117,4 +124,41 @@ function ScormStudioXBlock(runtime, element) {
     runtime.notify('cancel', {});
   });
 
+  $(element).find('input[name=display_type]').bind('click', function() {
+    showPopupLaunchType()
+  });
+
+  $(element).find('input[name=popup_launch_type]').bind('click', function() {
+    var display_type_popup = true;
+    showLaunchButtonTextField(display_type_popup)
+
+  });
+
+  function showPopupLaunchType() {
```

Comment:
Please change the name from `showPopupLaunchType()` to `showHidePopupLaunchOptions()` as the function shows as well as hides the options.